strapi/strapi
# Add bulk publish and unpublish modals

### What does it do?

- Add bulk publish and unpublish confirmation modals
- Create an injection zone for each one, in order to display the i18n text

I tried to keep it simple and not refactor code, but let me know if you think it could be necessary.

### Why is it needed?

To make sure users know what they're doing

### How to test it?

In the CM list view for a content type that has draft & publish:

- select 1 or more entry
- click publish all
- you should see a confirmation modal
- if the content type has i18n it should say something about it only applying to the current locale
- click on publish, it should log to console and close
- do the same for "unpublish all" button
